website/docs/developer/development-guide/code/versioning-releases.md

```diff
@@ -161,24 +204,33 @@ In addition to this, the dynamodb table and bucket version need to created as we
     --versioning-configuration Status=Enabled
 ```
 
-The EC2 key pair can be created using the AWS CLI. Make sure to save the generated private key for connecting to this instance later.
+Create an EC2 key pair can be created using the AWS CLI. Make sure to save the generated private key for connecting to this instance later.
 
 ```bash
-aws ec2 create-key-pair --key-name <PREFIX>
+aws ec2 create-key-pair --key-name <PREFIX> --query 'KeyMaterial' --output text > <PREFIX>.pem
 ```
 :::note
 Make sure your AWS is configured to use the cumulus sandbox account by using that account's AWS access keys before creating the EC2 key pair.
 :::
 
-For the `Deploy Dev Cumulus and ORCA Stack`  stage, add the following variables. 
+After hitting the play button on `Deploy Dev Cumulus and ORCA Stack`, but before hitting `Run` in the popup, replace the following variables with yours.
 
-- RDS_SECURITY_GROUP
-- RDS_USER_ACCESS_SECRET_ARN
-- DB_HOST_ENDPOINT
+- CUMULUS_AWS_ACCESS_KEY_ID
+- CUMULUS_AWS_SECRET_ACCESS_KEY
+- PREFIX
+- AWS_ACCOUNT_ID (for cumulus sandbox account)
+- AWS_SUBNET_ID1
+- AWS_SUBNET_ID2
+- VPC_ID
 - EARTHDATA_CLIENT_ID
 - EARTHDATA_CLIENT_PASSWORD
-- CUMULUS_ORCA_DEPLOY_TEMPLATE_VERSION
+- DB_ADMIN_PASSWORD
+- DB_USER_PASSWORD
+- DB_HOST_ENDPOINT
+- RDS_SECURITY_GROUP
+- RDS_USER_ACCESS_SECRET_ARN
+- CUMULUS_ORCA_DEPLOY_TEMPLATE_VERSION (optional)

```

Comment:
Since the Deploy Dev RDS Stack section mentions variables that were mentioned in the first Deploy DR ORCA Buckets section, I added every variable used by this stage. If we only want to call out variables that are newly set for each stage, I can strip some of them out.

Reply:
several of these variables will go away once orca-464 is done.